src/services/getVideos.js

```diff
@@ -0,0 +1,8 @@
+import { axiosSearch } from './axios';
+
+export const getVideos = (param, setListofVideos, setError) => {
+  axiosSearch
+    .get(`?part=snippet&maxResults=21&q=${param}&key=${process.env.REACT_APP_API_KEY}`)
+    .then((response) => setListofVideos(response.data.items))
+    .catch((e) => setError(e));
+};
```

Comment:
You have the fetching function in its own file, awesome, you could add some tests to make sure it works as expected